docs/tutorials/go.md

```diff
@@ -598,7 +598,7 @@ which connects to our server and send us transactions and other messages.
 
 Our application is almost ready to run, but first we'll need to populate the CometBFT configuration files.
 The following command will create a `cometbft-home` directory in your project and add a basic set of configuration files in `cometbft-home/config/`.
-For more information on what these files contain see [the configuration documentation](https://github.com/cometbft/cometbft/blob/v0.37.x/docs/core/configuration.md).
```

Comment:
The version used in the tutorial is the 0.37.0. That is why a full link was used here.

Reply:
I've reverted this change too